Project Management Process.md

```diff
@@ -0,0 +1,293 @@
+
+```
+Organization        |   Project 1           Project 2
+                    |  
+Roadmap             |   Roadmap             Roadmap  
+                    |  
+  Milestones        |     Milestone 1         Milestone A
+    Milestone 1     |       Goal 1              Goal A
+    Milestone 2     |       Goal 2            Milestone B
+    Milestone 3     |       Goal 3              Goal B
+    Milestone A     |     Milestone 2           Goal C
+    Milestone B     |       Goal 4              Goal D
+    Milestone C     |     Milestone 3      
+                    |       Goal 5      
+  Projects          |  
+    Project 1       |   Backlog             Backlog
+    Project 2       |     Goal 1              Goal A
+    Project 3       |     Goal 2              Goal B
+                    |     Goal 3              Goal C
+                    |     Goal 4              Goal D
+                    |     Goal 5
+```
+
+### Goals
+At the heart of each project are the goals. They are the basic unit of work. Goals can be anything that ***adds value to a release***. This includes issues reported by users, feature requests, bugs, Pull-Requests, general tasks such as refactoring, documentation or research. Goals are collected, maintained and tracked in the backlog by the project lead.
+
+A goal description includes the following:
+- Name / Summary
+  - Describes what the goal is on high level
+  - Example 1: "Reduce the size of the go-ipfs executable"
+  - Example 2: "js-ipfs should have API Documentation"
+- Description
+  - What the problem is and how to achieve this goal
+  - Should tell you how you will know that the problem has been solved or the goal has been achieved
+  - Describe the type of this goal: is it a bug? a new feature? something else?
+- Status
+  - The status should signal the goal's status towards completion
+  - What is the current state of this goal? Is it to-be-reviewed? In progress? Blocked? Done?
+  - A goal can be in only one state at a time
+- Dependencies (where applicable)
+  - Links to other goals that must happen before or concurrently with this one
+- Any additional information that helps to reach this goal (optional)
+
+TODO: a clear example
+
+### Backlog
+Backlog is the collection of all goals in a project. It should be as comprehensive as possible, including everything that needs to happen or should happen in a particular project. The backlog indicates priority by order: the items that have the highest priority are at the top of the list. The backlog is owned and maintained by the project lead. As part of daily work, the backlog gets updated and a general grooming should happen on weekly basis.
```

Comment:
From my perspective (and what I typically do and apparently similar to @haadcode's version) it is:

- Daily grooming - prepare the next day, I try to make sure to use my last breaths of each day to tell me what I need to do the next day, so that as soon as I get to work, I can start being efficient
- General grooming - 1 or 2 times a week, go through all the issues, make sure everyone is getting their answers, that duplicates are merged, that contributors have what their need in order to make contributions. A general 'keep the ball rolling for everyone'